app/components/datasources/datasources.js

```diff
@@ -213,6 +213,17 @@ export default translate()(class DataSources extends Component {
     )
   }
 
+  renderDataSourceWarningMessage(provider) {
+    if(provider.id === 'oauth/dexcom') {
+      return(
```

Comment:
Just a couple whitespace nitpicks here.  Should be space after the `if` and `return` keywords, and closing parentheses should line up with the `return`

Reply:
Thanks 👍-- need to figure out why my editor linting isn't catching that for me